pkg/node/sendoperation/buyrolls/buy_rolls.go

```diff
@@ -47,3 +49,23 @@ func (b *BuyRolls) Message() []byte {
 
 	return append(msg, buf[:nbBytes]...)
 }
+
+func DecodeMessage(data []byte) (*OperationDetails, error) {
+	buf := bytes.NewReader(data)
+
+	// Read operationId
```

Comment:
For consistency with sell_roll
```suggestion
	// Skip operationId
```